pages/linux/ethtool.md

```diff
@@ -0,0 +1,7 @@
+# ethtool
+
+> Displays and modifies Network Interface Controller (NIC) parameters.
+
+- Set the link speed, duplex, and speed autonegotiation:
```

Comment:
```suggestion
- Set the link speed, duplex mode, and parameter autonegotiation for a given interface:
```
Let's clarify here. Duplex "mode" is more correct, also the autonegotiation is not just for the speed. Let's also mention that this affects the selected interface only.